### DISCUSSION TITLE
Sensoren Primärstrom Verdichter und Spannung ; Konfiguration Thermostate?

### DISCUSSION THREAD

**Discussion by gorbi87** [Trunks1982/Daikin-Rotex-HPSU-CAN#56] [Q&A] [answered]
Hallo,

erstmal vielen Dank für das Super Projekt. Funktioniert einwandfrei.

Ist es möglich über den CAN Bus die Spannung sowie den Strom des Kompressors auszulesen um dadurch die aktuell elektrische Leistung zu berechnen? Dafür nutze ich zurzeit noch den espAltherma parallel. Dann könnte ich diesen in Rente schicken.

Zudem ist mir aktuell noch nicht ganz klar wie ich die Thermostate für Heizen/Kühlen aktivieren kann. Ich sehe zwar die Entitäten in HA aber als nicht verfügbar. Ich kann hier auch nichts in der ESP Konfiguration finden, wie z.B. verwendete GPIOs. Vielen Dank

**Comment by Trunks1982** [comment, marked as answer]
Leider nicht möglich über Canbus Stromwerte abzufragen.Dafür würde ich aber auch zb einen Shelly 3em empfehlen. Damit kann man jede einzelne Phase abfragen.

die Schalter für heizen und Kühlen kann man nur mitttels relais schalten. Dafür gibt es extra Komponenten für den Esp. Oder was hier auch einige schon einsetzen das DIY Board. Da sind die schon vorhanden.

**Comment by gorbi87** [comment]
Danke für die schnelle Antwort. Gut dann schaue ich mal ob ich diesen Teil  weiter über den ESP laufen lasse oder eventl. einen Shelly.

Das mit den Relais ist mir bewusst, diese sind auch schon angeschlossen. Ich frage mich nur wo ich diese in der Konfiguration aktivieren kann bzw. wo die Zuordnung Relais Heizen und Relais Kühlen zum jeweiligen GPIO stattfindet sodass diese auch in der Weboberfläche auftauchen

**Reply by Trunks1982** [reply]
In der full.yaml stehen die GPIO's drin kann man natürlich ändern wenn man mag. Aktivieren kann man sie wenn man vorne die # Zeichen löscht.

**Reply by gorbi87** [reply]
in der aktuellen main Branch? Dann bin ich glaub ich blind :) sehe da nur die GPIOs für den CAN Bus...

**Reply by Trunks1982** [reply]
Ah entschuldige wird mit der nächsten Version kommen.

**Reply by gorbi87** [reply]
Perfekt. Vielen Dank

**Reply by wrfz** [reply]
Neue version **2024.09.6** ist da: https://github.com/Trunks1982/Daikin-Rotex-HPSU-CAN/releases
